en/streaming-integrator/docs/develop/installing-siddhi-extensions.md

```diff
@@ -3,203 +3,130 @@
+The dialog box displays all the dependencies that need to be manually installed. For each dependency, the dialog box provides the following information.
+
+- **Instructions** to download (and depending on the dependency, to convert) the JAR of the dependency.
+
+- **Installation Location** where the downloaded JAR (and depending on the dependency, the converted OSGi bundle) needs to be placed in order to install the dependency. The following table especifies the directory in which you need to place the JAR/OSGi bundle depending on the installation location.
+
+    |**Installation Location**|**Directory**|
+    |---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+    |**bundle in runtime**|Place the OSGi bundle(s) you downloaded/converted in either the `<SI_HOME>/lib` or the `<SI_HOME>/bundles` directory based on the instructions.|
+    |**jar in runtime**   |Place the non-OSGi bundle(s) you downloaded in the `<SI_HOME>/jars` directory.                                                                 |
+    |**jar in samples**   |Place the non-OSGi bundle(s) you dowloaded in the `<SI_HOME>/samples/sample-clients/lib` directory.                                            |
 
 
 ## Configuring Extension Dependencies
 
-Configurations of extensions are loaded from the configuration file, which is located in `<SI_HOME>/wso2/server/resources/extensionsInstaller/extensionDependencies.json`.
+Configurations of extensions are loaded from the `<SI_HOME>/wso2/server/resources/extensionsInstaller/extensionDependencies.json` configuration file.
 
-When you are working with [custom extensions](../admin/writing-Custom-Siddhi-Extensions.md#writing-custom-siddhi-extensions), and if you want a custom extension to be installable from the Extension Installer, you should add the extension's configuration to the configuration file.
+When you are working with [custom extensions](../admin/writing-Custom-Siddhi-Extensions.md#writing-custom-siddhi-extensions), and if you want a custom extension to be installable from the Extension Installer, you need to add the configuration of the extension to this configuration file.
 
-An extension's configuration is a JSON object that looks like the following.
+The configuration of an extension is a JSON object that looks as follows:
 
 ```json
-"<extension_name>": {
-  "extension": {...},
-  "dependencies": [
-    {...},
-    {...}
-  ]
-}
+    "<extension_name>": {
+      "extension": {...},
+      "dependencies": [
+        {...},
+        {...}
+      ]
+    }
 ```
-`<extension_name>` - which is the key of this JSON object, is the uniquely identifiable name of the extension. This is same as [`extension`](#extension)`.name`.
+
+`<extension_name>` which is the key of this JSON object, is the uniquely identifiable name of the extension. The extension is described under [`extension`](#extension)`.
```

Comment:
Please remove the **`** before the last full stop.

Reply:
Done